test/e2e/lib/gutenberg/tracking/general-tests.js

```diff
@@ -122,6 +122,41 @@ export function createGeneralTests( { it, editorType, postType } ) {
 		assert.strictEqual( toggleEvents[ 1 ][ 1 ].is_open, true );
 	} );
 
+	it( 'Tracks "wpcom_block_editor_list_view_select" event', async function () {
+		const editor = await EditorComponent.Expect( this.driver, gutenbergEditorType );
+
+		// The list view toggle button is not available on mobile
+		if ( editor.screenSize === 'mobile' ) {
+			return this.skip();
+		}
+
+		await editor.addBlock( 'Columns' );
```

Comment:
It doesnt look like we are clicking or checking this this block, do we even need to add it?

Reply:
Nope!